_infra/helm/uaa/templates/deployment.yaml

```diff
@@ -125,7 +125,7 @@ spec:
               value: "jdbc:postgresql://$(DB_HOST):$(DB_PORT)/$(DB_NAME)?sslmode=disable"
               {{- end }}
             - name: JAVA_OPTS
-              value: "-Dspring.profiles.active=postgresql -Dlog4j.configurationFile=file:/etc/config/log4j2.properties"
+              value: "-Dspring.profiles.active=postgresql -Dlog4j.configurationFile=file:/etc/config/log4j2.properties -Dlogging.config=/etc/config/log4j2.properties"
```

Comment:
Can we delete the `Dlog4j.configurationFile` setting here if we've got the one you added?